modules/gltf/gltf_document.cpp

```diff
@@ -3931,6 +3931,35 @@ Error GLTFDocument::_parse_materials(Ref<GLTFState> p_state) {
 			}
 		}
 
+		// Supersedes KHR_materials_pbrSpecularGlossiness
+		// see https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_specular/README.md
+		if (material_extensions.has("KHR_materials_specular")) {
+			Dictionary specular_properties = material_extensions["KHR_materials_specular"];
+			if (specular_properties.has("specularFactor")) {
+				float specular_value = specular_properties["specularFactor"];
+				material->set_specular(specular_value / 2.0);
+
+			} else if (specular_properties.has("specularColorFactor")) {
+				const Array &arr = specular_properties["specularColorFactor"];
+				ERR_FAIL_COND_V(arr.size() != 3, ERR_PARSE_ERROR);
+				float specular_value = arr[0];
+				if (specular_value > 2.0) {
+					specular_value = 2.0;
+				}
+				// we don't have support for specular tint color, but there's no way
+				// of knowing just the specular factor if the tint color is not (1,1,1)
+				// Therefore, enforce the constraint that all values must be equal
+				// However we can't know if (0.5, 0.5, 0.5) is a grey tint * specular factor of 1
+				// or a specular factor of 0.5. That's up to the user to set the Blender model
+				// correctly for Godot usage
+				if ((arr[0] == arr[1]) && (arr[1] == arr[2])) {
+					material->set_specular(specular_value / 2.0);
```

Comment:
@clayjohn Is it possible allow 3 channel specular?

Reply:
Should a change to the scene shader be part of this PR or separate?

Reply:
That should definitely be a separate discussion. Godot doesn't support `specular_tint` right now.

Reply:
@clayjohn Do you know where 3 channel specular is being tracked?